.github/workflows/readthedocs-pr-links.yaml

```diff
@@ -0,0 +1,16 @@
+name: Read the Docs Pull Request Preview
+on:
+  pull_request_target:
```

Comment:
Had to [read the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) on why this was chosen vs simply `pull_request`. Summary for others' benefit:

> This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow. This event allows your workflow to do things like label or comment on pull requests from forks.

Nice work!